src/Gui/NavigationStyle.cpp

```diff
@@ -797,6 +798,13 @@ void NavigationStyle::doZoom(SoCamera* camera, SbBool forward, const SbVec2f& po
 //    }
 }
 
+void NavigationStyle::doZoom_wheel(SoCamera* camera, int wheeldelta, const SbVec2f& pos)
```

Comment:
Is this helper function still needed? It looks like it's only used in one location now, and as it's only 5 lines, it could probably just be implemented directly where needed.